server-phoenix/lib/helios/slack_channel_names.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Helios.SlackChannelNames do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+  import Ecto.Query
+
+  schema "slack_channel_names" do
+    field :channel_name, :string
+    field :channel_id, :string
+
+    timestamps()
+  end
+
+  def with_id(query, channel_id) do
```

Comment:
Might make this easier to use to provide a default value for `query`

```suggestion
  def with_id(query \\ __MODULE__, channel_id) do
```